src/avgScrabbleScoreFromFile.js

```diff
@@ -12,11 +12,20 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  */
 function avgScrabbleScoreFromFile(fileName) {
   // This is your job. :)
+  let text = fs.readFileSync(`${fileName}`, 'utf-8');
```

Comment:
1. No reason to do `${fileName}` if `fileName` is already a string. `fs.readFileSync(fileName, 'utf-8')` will work.
2. You can use `trimRight()` to remove trailing whitespace.

If you want to handle newline characters in a cross-platform way with `split`:

- https://www.google.com/search?q=node+cross+platform+newline+split
- https://stackoverflow.com/questions/21895233/how-in-node-to-split-string-by-newline-n/21895299